testsuite/integration/basic/src/test/java/org/wildfly/test/integration/observability/opentelemetry/OpenTelemetrySetupTask.java

```diff
@@ -11,30 +11,38 @@
 import org.jboss.dmr.ModelNode;
 
 public class OpenTelemetrySetupTask implements ServerSetupTask {
+    private static boolean initialized = false;
+    private static int count = 2; // Ugly hack to prevent multiple reloads
```

Comment:
If this is just to avoid reloads, it adds complexity and fragility. Test classes should clean up after themselves so the next test is starting from a known config, and not count on some other test class to do it at some later point.

Reply:
It is and it does, but if you're ok with the reloads, I'll yank this like I'm starting a mower.

Reply:
Pushed.